components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -116,6 +116,117 @@ std::string URLMethodToRequestType(ledger::type::UrlMethod method) {
   }
 }
 
+bool ResetOnFileTaskRunner(
```

Comment:
Let's name this something a little more self-documenting; maybe `DeleteFilesOnFileTaskRunner`?

Reply:
Done.